Indicators/Momentum.cs

```diff
@@ -26,6 +26,11 @@ public class Momentum : WindowIndicator<IndicatorDataPoint>, IIndicatorWarmUpPer
         /// </summary>
         public int WarmUpPeriod => Period;
 
+        /// <summary>
+        /// Gets a flag indicating when this indicator is ready and fully initialized
+        /// </summary>
+        public override bool IsReady => Samples > WarmUpPeriod;
```

Comment:
If the indicator is ready when `Samples` is greater than `WarmUpPeriod`, it means that `WarmUpPeriod` is incorrect, because we need `WarmUpPeriod + 1` to be ready.
```csharp
public int WarmUpPeriod => Period + 1;
public override bool IsReady => Samples > Period;
```